### DISCUSSION TITLE
Lesson 13: chainId from useEthers not returning any value

### DISCUSSION THREAD

**Discussion by Python-David** [smartcontractkit/full-blockchain-solidity-course-py#1739] [Q&A] [answered]
When I console.log I keep getting undefined and "dev", a clear indication that what is happening is the else part of this function: const networkName = chainId ? helperConfig[chainId] : "dev"

I need help with this please, thanks.

**Comment by cromewar** [comment, marked as answer]
Hello @Python-David 
The chain id now should use 5 for Goerli due to the recent changes for etheruem.
However the lesson 13 was archived as is too outdated, pleas you can refer to the Js version for a full front end implementation and skip this one.
